index.html

```diff
@@ -13,14 +13,22 @@
 
 <base target="_blank">
 
+<meta property="og:type" content="website">
+<meta property="og:title" content="Delta Chat Cosmos">
+<meta property="og:image" content="https://cosomos.delta.chat/social.png">
+<meta property="og:url" content="https://cosomos.delta.chat">
+
+<meta name="twitter:card" content="summary_large_image">
+<meta property="twitter:site" content="@delta_chat">
+
 <style>
 body {
   margin: 0px;
   padding: 0px;
   font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
   background-color: #000;
   color: #bbb;
-  font-size: 16px;
+  font-size: 15px;
```

Comment:
i would stay with the larger option of 16px here, this is at least better for my older eyes :)

but no blocker, also 15px is okayish :)

Reply:
For me it goes a little too wide (sometimes it's a quirk of certain pixel values, like if you observe between 10px to 12px how it changes). If it's not a blocker, I'll keep it.